mlflow/server/js/src/experiment-tracking/components/experiment-page/utils/experimentPage.column-utils.ts

```diff
@@ -214,41 +214,43 @@ export const useRunsColumnDefinitions = ({
       resizable: false,
     });
 
-    // Date and expander selection column
+    // Run name column
     columns.push({
-      headerName: ATTRIBUTE_COLUMN_LABELS.DATE,
-      headerTooltip: ATTRIBUTE_COLUMN_SORT_KEY.DATE,
+      headerName: ATTRIBUTE_COLUMN_LABELS.RUN_NAME,
+      colId: TAGS_TO_COLUMNS_MAP[ATTRIBUTE_COLUMN_SORT_KEY.RUN_NAME],
+      headerTooltip: ATTRIBUTE_COLUMN_SORT_KEY.RUN_NAME,
       pinned: 'left',
+      sortable: true,
       field: 'runDateAndNestInfo',
-      initialWidth: 150,
-      cellRenderer: 'DateCellRenderer',
+      cellRenderer: 'RunNameCellRenderer',
       cellRendererParams: { onExpand },
       equals: (dateInfo1, dateInfo2) => isEqual(dateInfo1, dateInfo2),
-      sortable: true,
       headerComponentParams: {
         ...commonSortOrderProps,
-        canonicalSortKey: ATTRIBUTE_COLUMN_SORT_KEY.DATE,
+        canonicalSortKey: ATTRIBUTE_COLUMN_SORT_KEY.RUN_NAME,
         getClassName: getHeaderClassName,
       },
       cellClass: getCellClassName,
+      initialWidth: 260,
     });
 
-    // Run name column
+    // Date and expander selection column
     columns.push({
-      headerName: ATTRIBUTE_COLUMN_LABELS.RUN_NAME,
-      colId: TAGS_TO_COLUMNS_MAP[ATTRIBUTE_COLUMN_SORT_KEY.RUN_NAME],
-      headerTooltip: ATTRIBUTE_COLUMN_SORT_KEY.RUN_NAME,
+      headerName: ATTRIBUTE_COLUMN_LABELS.DATE,
+      headerTooltip: ATTRIBUTE_COLUMN_SORT_KEY.DATE,
       pinned: 'left',
-      field: 'runName',
       sortable: true,
-      cellRenderer: 'RunNameCellRenderer',
+      field: 'runDateAndNestInfo',
+      cellRenderer: 'DateCellRenderer',
+      cellRendererParams: { onExpand },
+      equals: (dateInfo1, dateInfo2) => isEqual(dateInfo1, dateInfo2),
```

Comment:
```suggestion
      equals: isEqual,
```

Can we directly pass `isEqual`?